sources/light_client.move

```diff
@@ -83,8 +82,8 @@ public(package) fun new_light_client_with_params(params: Params, start_height: u
 }
 
 
-// Helper function to initialize new light client.
-// network: 0 = mainnet, 1 = testnet
+/// Helper function to initialize new light client.
+/// network: 0 = mainnet, 1 = testnet
 public fun new_light_client(
```

Comment:
Could we move this to entry function?

Reply:
I was thinking about it as well, but we can't - there is an error

Reply:
Ah yes, I remember this.